core/shortcut_items.js

```diff
@@ -129,16 +129,18 @@ const registerCut = function() {
     name: names.CUT,
     preconditionFn: function(workspace) {
       return !workspace.options.readOnly && !Gesture.inProgress() &&
-          Blockly.selected && Blockly.selected.isDeletable() &&
-          Blockly.selected.isMovable() && !Blockly.selected.workspace.isFlyout;
+          common.getSelected() && common.getSelected().isDeletable() &&
+          common.getSelected().isMovable() &&
+          !common.getSelected().workspace.isFlyout;
```

Comment:
OK: maybe *this* is the point where we do that?

Reply:
Updated